src/main/scala/org/constellation/util/HealthChecker.scala

```diff
@@ -94,7 +94,17 @@ class HealthChecker[F[_]: Concurrent](
         .map(_.filter { case (_, pd) => NodeState.isNotOffline(pd.peerMetadata.nodeState) })
       nextSnapshotHeights <- peers.values.toList
         .map(_.peerMetadata.toPeerClientMetadata)
-        .traverse(apiClient.snapshot.getNextSnapshotHeight().run)
+        .traverse(
+          a =>
+            apiClient.snapshot
+              .getNextSnapshotHeight()
+              .run(a)
+              .map {
+                case (id, height) => (id, height)
```

Comment:
is that part needed?

Reply:
Yes, in case of incorrect request, we can see it in logs on mainnet

Reply:
Huh, wait

Reply:
The map itself is useless, I’ll remove it